plugins/net_plugin/net_plugin.cpp

```diff
@@ -1373,6 +1374,10 @@ namespace eosio {
       stat.syncing = peer_syncing_from_us;
       stat.is_bp_peer = is_bp_connection;
       stat.is_socket_open = socket_is_open();
+      stat.is_blocks_only = is_blocks_only_connection();
+      stat.is_transactions_only = is_transactions_only_connection();
+      elog("last vote ${v}", ("v", last_vote_received.load()));
```

Comment:
why is this elog here?

Reply:
Thanks! I had missed putting it in FC_REFLECT and I was debugging. I'll remove it.